web/styles/subscriptions.css

```diff
@@ -664,6 +674,31 @@ h4.user_group_setting_subsection_title {
     width: 100%;
 }
 
+.left .group-name .fa-ban,
+.right .group-name .fa-ban {
+    scale: 0.8;
```

Comment:
This is the way we scale icons in the codebase:

``` diff
diff --git a/web/styles/subscriptions.css b/web/styles/subscriptions.css
index 0a307db958..095416c392 100644
--- a/web/styles/subscriptions.css
+++ b/web/styles/subscriptions.css
@@ -676,7 +676,7 @@ h4.user_group_setting_subsection_title {
 
 .left .group-name .fa-ban,
 .right .group-name .fa-ban {
-    scale: 0.8;
+    font-size: 0.8em;
     opacity: 0.6;
 }
 
```

`git grep scale: web/styles` would be a good way to determine this was not a standard technique for us.